pkg/configmap/configmap.go

```diff
@@ -42,12 +41,16 @@ func ValidateOperatorConfigMap(configmap OperatorConfigMap) error {
 func GetOperatorConfigMap(kubeClient client.Client) (OperatorConfigMap, error) {
 	var OperatorConfigMap OperatorConfigMap
```

Comment:
btw, why does this variable name start with a capital o?

Reply:
Nice catch!